vtex/billing-app
# Fix/international

#### What is the purpose of this pull request?
Make app purchases only available to VTEX stores in Brazil

#### What problem is this solving?
Payments for apps by stores outside Brazil are yet to be defined. While there is no consensus or fixed process for their payment, the billing app will prevent those stores from contracting **paid apps**

#### How should this be manually tested?
1. Clone/pull this repo/branch
2. Link this app in an international test account, such as `qastorecharlie`
3. Try to install a paid app, such as `compasso.correioslog`, and a free app, such as `vtex.instore`, either by using the toolbelt or the apps admin. The former should throw a `area_unavailable` code, and the latter should be successfull

#### Types of changes
